### PR TITLE
Update preface.rst

### PR DESCRIPTION
**Pull request by ProductRyan**
Add a description of your changes with links to any relevant material.
- updating basic info and links

### Date Needed (optional)
- N/A

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers
- @sarina 

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:.

- [ ] Subject matter expert:
- [ ] Subject matter expert:
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support:
- [ ] PM review:

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/preface.rst [hunk 1]
```diff
@@ -36,8 +36,7 @@ Documentation
 ==============
 
 The `edX Help Center for Learners`_ includes topics to help you understand how
-to use the edX learning management system. The Help Center is also available 
-when you select **Help** while you are in a course, and from your edX dashboard.
+to use the edX learning management system. The Help Center is also available from withing the learning experinece by selecting the **Help** menu and from within your edX dashboard.
```
Comment (sarina) [inline en_us/shared/preface.rst]: Typo: "withing"

The last part of this sentence reads awkwardly to me. You use "within" twice. Would it be accurate to say,

"The Help Center is also available from within the learning experience. From your edX dashboard, select the **Help** menu."

File: en_us/shared/preface.rst [hunk 2]
```diff
@@ -118,33 +114,22 @@ Course Team Support in the edX Partner Portal
...
 ***********************
-The Open edX Portal
+The Open edX Site
 ***********************
 
-The `Open edX Portal`_ is the destination for learning about hosting an Open
-edX instance, extending the edX platform, and contributing to Open edX. In
-addition, the Open edX Portal provides product announcements and other
+The `Open edX Site`_ is the destination for learning about hosting an Open
+edX instance, extending the edX platform, and contributing to the Open edX project. In
```
Comment (sarina) [inline en_us/shared/preface.rst]: Can you say "Open edX platform"?

File: en_us/shared/preface.rst [hunk 3]
```diff
@@ -118,33 +114,22 @@ Course Team Support in the edX Partner Portal
...
+addition, the Open edX Psite provides product announcements and other
```
Comment (sarina) [inline en_us/shared/preface.rst]: Typo: Psite

**Review by sarina — CHANGES_REQUESTED** [review]
Hey @ProductRyan this is great! Could you take a look at my comments and address them?

I'm not sure if you need it, but here's instructions on modifying your pull request: https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html

File: en_us/shared/preface.rst [hunk 4]
```diff
@@ -118,33 +114,22 @@ Course Team Support in the edX Partner Portal
...
 
 ***********************
-The Open edX Portal
+The Open edX Site
 ***********************
 
-The `Open edX Portal`_ is the destination for learning about hosting an Open
-edX instance, extending the edX platform, and contributing to Open edX. In
-addition, the Open edX Portal provides product announcements and other
+The `Open edX Site`_ is the destination for learning about hosting an Open
```
Comment (sarina) [inline en_us/shared/preface.rst]: Ah, this breaks the link that was formerly called "Open edX Portal". This is called Open edX Portal throughout this document (multiple times). So I wouldn't make this change right now; if you want to change it everywhere in this doc I'd make a new pull request.

**Comment by sarina** [comment]
Hey @ProductRyan  - do you have time/capacity to address the changes?

**Comment by feanil** [comment]
@ProductRyan checking back in, do you have time to address the change request?

File: en_us/shared/preface.rst (hunk 3 above)
Comment (sarina) [inline en_us/shared/preface.rst]:
```suggestion
addition, the Open edX site provides product announcements and other
```

File: en_us/shared/preface.rst (hunk 2 above)
Comment (sarina) [inline en_us/shared/preface.rst]:
```suggestion
edX instance, extending the Open edX platform, and contributing to the Open edX project. In
```

File: en_us/shared/preface.rst (hunk 4 above)
Comment (sarina) [inline en_us/shared/preface.rst]:
```suggestion
The `Open edX Portal`_ is the destination for learning about hosting an Open
```

File: en_us/shared/preface.rst (hunk 1 above)
Comment (sarina) [inline en_us/shared/preface.rst]:
```suggestion
to use the Open edX learning management system. On edx.org, the Help Center is also available from within the learning experience by selecting the **Help** menu from within your edX dashboard.
```

**Review by sarina — CHANGES_REQUESTED** [review]

**Review by sarina — APPROVED** [review]

**Pull request merged**